bindings/rust/wasmedge-sys/build.rs

```diff
@@ -41,6 +41,8 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         .clang_arg(format!("-I{}", inc_dir.as_path().display()))
         .prepend_enum_name(false) // The API already prepends the name.
         .dynamic_link_require_all(true)
+        .derive_debug(true)
```

Comment:
@apepkuss 
Could you please help check this part? Will this change any behavior of our current wasmedge-sys crate?

Reply:
@apepkuss Does the rust side change look good to you?